packages/api/cms-api/src/user-permissions/dto/user-permission.input.ts

```diff
@@ -23,7 +23,7 @@ export class UserPermissionOverrideContentScopesInput {
 
 @InputType()
 export class UserPermissionInput {
-    @Field(() => String)
+    @Field(() => CombinedPermission)
```

Comment:
CombinedPermission is defined as const, does this only work because of registerEnumType?

Reply:
> CombinedPermission is defined as const, does this only work because of registerEnumType?

Yes, I assume that Nest only uses the object's reference here (maybe that's why we need a registerEnumType call in the first place).